linera-core/src/unit_tests/wasm_client_tests.rs

```diff
@@ -148,6 +158,18 @@ where
     let balance_after_init = creator.local_balance().await?;
     assert!(balance_after_init < balance_after_messaging);
 
+    let large_bytecode = Bytecode::new(vec![0; policy.maximum_bytecode_size as usize + 1]);
+    let small_bytecode = Bytecode::new(vec![]);
+    // Publishing bytecode that exceeds the limit fails.
+    let result = publisher
+        .publish_bytecode(large_bytecode.clone(), small_bytecode.clone())
+        .await;
+    assert_matches!(result, Err(ChainClientError::LocalNodeError(_)));
+    let result = publisher
+        .publish_bytecode(small_bytecode, large_bytecode)
+        .await;
+    assert_matches!(result, Err(ChainClientError::LocalNodeError(_)));
```

Comment:
Is it possible to verify that it's the _exact_ error variant we expect? This is then kindof a test for the client-facing API (i.e. the errors we return should be documented).

Reply:
Yes, I should fill these in. It's just a bit ugly because of a lot of nesting, some of which even involves boxes, I think.